task-57941: Fix the title in SpaceInfos Portlet is positioned on the center

IlhemEssaadi opened the pull request:
FIX : The problem is that the title is centered and is positioned by the padding of the CSS class spaceInfosApp, fixed by moving the title outside this class and adding  vuetify spacing Property.

ahamdi commented on webapp/portlet/src/main/webapp/vue-apps/space-infos-app/components/ExoSpaceInfos.vue:
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app>
-    <div class="body-1 text-uppercase text-sub-title">{{ $t("social.space.description.title") }}</div>
+    <div class="body-1 text-uppercase text-sub-title mx-n1">{{ $t("social.space.description.title") }}</div>
```
Could you check where the margin is added and fix it from there instead of adding a negative margin ?

montassarhamadi committed:
TASK-57941: Improve design of SpaceInfos Portlet.
Problems:
1-centralized title
2-duplicated application id `spaceInfosApp`
3-bad display of list managers and redactors of the space

Fix:
1-remove `center` class from `div` of the title.
2- remove duplicated id `spaceInfosApp` in component `ExoSpaceInfos.vue` and fix padding for this id selector in module platform-ui.
3-fix padding left and right for list of managers and redactors.
